eqcorrscan/core/match_filter/family.py

```diff
@@ -503,8 +503,8 @@ def write(self, filename, format='tar'):
     def lag_calc(self, stream, pre_processed, shift_len=0.2, min_cc=0.4,
                  horizontal_chans=['E', 'N', '1', '2'], vertical_chans=['Z'],
                  cores=1, interpolate=False, plot=False, plotdir=None,
-                 parallel=True, process_cores=None, ignore_length=False,
-                 ignore_bad_data=False, **kwargs):
+                 npy=False, npydir=None, parallel=True, process_cores=None,
```

Comment:
Same arg comment as above, and please do not change the order of the arguments just in case someone is relying on the order of the args.